packages/main/src/Tokenizer.ts

```diff
@@ -251,6 +276,10 @@ class Tokenizer extends UI5Element {
 		const target = e.target as Token;
 		if (!e.detail) { // if there are no details, the event is triggered by a click
```

Comment:
If you have one long token and another one, so the long token is expanded. Trying to delete the long token from its clear icon it is not possible, it is just scrolled to the beginning, _delete is not invoked at all. In order to be able to delete it from its clear icon, it should be the only token in the ui5-multi-input/combobox.